src/main/java/seedu/address/model/goodsreceipt/GoodsReceiptUtil.java

```diff
@@ -2,31 +2,17 @@
 
 import java.util.List;
 
-import seedu.address.model.person.Name;
-
 /**
  * Contains a set of utility methods for goods.
  */
 public class GoodsReceiptUtil {
     /**
-     * Return a list of goods belonging to a specific supplier.
-     *
-     * @param supplier A valid supplier name.
-     * @param goodsReceipt A list of goodsReceipts.
-     */
-    public static List<GoodsReceipt> filterGoodsReceiptsBySuppliers(Name supplier, List<GoodsReceipt> goodsReceipt) {
-        return goodsReceipt.stream().filter(g -> g.isFromSupplier(supplier)).toList();
-    }
-
-    /**
-     * Return the total quantity of goods belonging to a specific supplier.
+     * Return the total quantity of goods from a list of goodsReceipts.
      *
-     * @param supplier A valid supplier name.
      * @param goodsReceipt A list of goodsReceipts.
      */
-    public static int sumQuantityBySuppliers(Name supplier, List<GoodsReceipt> goodsReceipt) {
-        return goodsReceipt.stream().filter(g -> g.isFromSupplier(supplier))
-                                 .map(GoodsReceipt::getQuantity)
+    public static int sumQuantity(List<GoodsReceipt> goodsReceipt) {
+        return goodsReceipt.stream().map(GoodsReceipt::getQuantity)
                                  .reduce(Integer::sum)
                                  .orElse(0);
```

Comment:
```suggestion
                                 .reduce(0, Integer::sum)
```
either way works fine